### PR TITLE
Added anchr.io and http2-serverpush-proxy.

### PR DESCRIPTION
**Pull request by n1try**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines. 
        
    ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language```
- [x] Your additions are ordered alphabetically.
- [x] Your additions are [free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md).
- [x] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant issues or PRs.
- [x] Any category you are creating has the minimum requirement of 3 items.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -175,6 +175,7 @@ See also [Static Site Generators](#static-site-generators), [Content Management
   * [saveto.co](https://saveto.co/) - An open source, home for the best links on the web. ([Source Code](https://github.com/saveto-co/saveto)) `MIT` `Nodejs`
   * [Shaarli](https://github.com/shaarli/Shaarli) - Personal, minimalist, super-fast, no-database bookmarking and link sharing plaform. ([Demo](http://shaarlidemo.tuxfamily.org/#demo)) `Zlib` `PHP`
   * [unmark](https://unmark.it/) - An open source to do app for links. ([Source Code](https://github.com/plainmade/unmark)) `MIT` `PHP`
+  * [Anchr](https://anchr.io) - Anchr is a toolbox for tiny tasks on the internet, including bookmark collections, URL shortening and (encrypted) image uploads. ([Source Code](https://github.com/n1try/anchr)) `GPLv3` `Nodejs`
```
Comment (nodiscc) [inline README.md]: Looks like a cool project. Can you sort the section alphabetically? Thanks

File: README.md [hunk 2]
```diff
@@ -903,7 +905,8 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
   * [Swiperproxy](https://swiperproxy.github.io/) - A lightning-fast, open source web proxy that is easy for you to run and customize. ([Demo](http://proxy.donttrustrobots.nl/), [Source Code](https://github.com/swiperproxy/swiperproxy)) `MIT` `Python`
   * [Tinyproxy](https://banu.com/tinyproxy/) - Light-weight HTTP/HTTPS proxy daemon. ([Source Code](https://github.com/tinyproxy/tinyproxy)) `GPLv2` `C`
   * [Traefik](https://traefik.io/) - Træfɪk is a modern HTTP reverse proxy and load balancer made to deploy microservices with ease. It supports several backends (Docker, Swarm, Mesos/Marathon, …) to manage its configuration automatically and dynamically. ([Source Code](https://github.com/containous/traefik)) `MIT` `Go`
-
+  * [http2-serverpush-proxy](https://github.com/n1try/http2-serverpush-proxy) -  A reverse proxy that helps to automatically utilize HTTP/2.0's server push mechanism for static websites. Available as middleware and standalone application. ([Source Code](https://github.com/n1try/http2-serverpush-proxy)) `MIT` `Nodejs`
```
Comment (nodiscc) [inline README.md]: Pleas sort the section alphabetically.

**Comment by n1try** [comment]
Thanks for feedback and btw. also thanks for this giant list!

**Comment by Kickball** [comment]
Replaced by #859 as @n1try has deleted his branch.

**Pull request closed without merging**